search/django/weedid/views.py

```diff
@@ -122,8 +197,8 @@ def update_categories(request):
         validate(updated_weedcoco_json, schema="compatible-coco")
     except JsonValidationError as e:
         return HttpResponseBadRequest(e.message)
-    except Exception:
-        return HttpResponseNotAllowed("Failed to update categories")
+    except Exception as e:
+        return HttpResponseBadRequest(str(e))
```

Comment:
please log the exception with print_exc (or just leave the exception unhandled and let it return 50x)